src/app/app.server.ts

```diff
@@ -0,0 +1,16 @@
+import express, { Express } from "express";
+import { router as AppRouter } from "./app.router";
```

Comment:
You should name the `AppRouter` to `appRouter` since is a variable and not a Class. In TypeScript, we follow the JavaScript style guides

Reply:
Fixed, thanks @nestorvanz :3